fvm/crypto/crypto.go

```diff
@@ -37,12 +37,26 @@ func (DefaultSignatureVerifier) Verify(
 	publicKey crypto.PublicKey,
 	hashAlgo hash.HashingAlgorithm,
 ) (bool, error) {
-	hasher := NewHasher(hashAlgo)
-	if hasher == nil {
+	var hasher hash.Hasher
+
+	switch hashAlgo {
+	case hash.SHA2_256:
+		hasher = hash.NewSHA2_256()
+	case hash.SHA3_256:
+		hasher = hash.NewSHA3_256()
+	case hash.SHA2_384:
+		hasher = hash.NewSHA2_384()
+	case hash.SHA3_384:
+		hasher = hash.NewSHA3_384()
+	case hash.KMAC128:
```

Comment:
Signature verification should support KMAC only when we're supporting BLS, but I guess that doesn't matter because we're on a feature branch and we will merge everything together to master.

Reply:
I can add this in the https://github.com/onflow/flow-go/pull/768 PR